Sming/Services/WebHelpers/base64.cpp

```diff
@@ -1,115 +1,70 @@
-/* base64.c : base-64 / MIME encode/decode */
-/* PUBLIC DOMAIN - Jon Mayo - November 13, 2003 */
-/* $Id: base64.c 156 2007-07-12 23:29:10Z orange $ */
+/*
```

Comment:
Please replace the beginning comment with the standard comment that we have. Something like this will be enough:

```c
/****
 * Sming Framework Project - Open Source framework for high efficiency native ESP8266 development.
 * Created 2015 by Skurydin Alexey
 * http://github.com/SmingHub/Sming
 * All files of the Sming Core are provided under the LGPL v3 license.
 *
 *
 * @author: 2018 - Mikee47 <your@email>
 *
 ****/
```